.travis.yml

```diff
@@ -5,11 +5,16 @@ services:
 python:
 - 3.4
 install:
-- pip install -e .
-- pip install pyyaml
+# Make a wheel and install it to test to catch possible
+# issues with MANIFEST.in
+- pip install --no-cache-dir pyyaml pytest wheel
+- python setup.py bdist_wheel
+- pip install dist/*.whl
 
 script:
-  - pytest -s -v tests/${REPO_TYPE}
+  # cd into tests so CWD being repo2docker does not hide
+  # possible issues with MANIFEST.in
+  - cd tests && pytest -s -v ${REPO_TYPE}
```

Comment:
is there some magic we can use to only show stdout/err when a test fails? As is you get a lot of output which is a bit annoying when trying to find the bits related to the error :-/

Reply:
I agree! We should probably do that in a different PR / Issue though!